content/storage/postgres/docker-compose-for-postgres.md

```diff
@@ -88,24 +95,29 @@ COMPOSE_PROJECT_NAME=db-compose
 tree . -a
 .
 ├── .env
-├── docker-compose.yml
+├── docker-compose.[yml / yaml]
 └── init
     └── 1.create-user-table.sql
 ```
 
 #  Useful Docker compose commands
-- To launch a container.
+- To launch a container
 ```sh
 docker-compose up
 ```
 
-- To remove a container with its volumes.
+- To launch a container as a daemon
+```sh
+docker-compose up -d
+```
+
+- To remove a container with its volumes
 ```sh
 docker-compose down --volumes
 ```
 
 # Connect a database server
-- Use these value to connect to a server server
+- Use these value to connect to a database server (Postgres)
```

Comment:
@dmakeroam  Thank you I want to say a database server but my typo to repeat myself.

Reply:
OK